<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog1DScale</class>
 <widget class="QDialog" name="Dialog1DScale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interval Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupX">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>x-range</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioAutoCalcX">
        <property name="text">
         <string>auto</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioManualX">
        <property name="text">
         <string>manual adjustment:</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>from</string>
          </property>
          <property name="buddy">
           <cstring>doubleSpinMinX</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinMinX">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>to</string>
          </property>
          <property name="buddy">
           <cstring>doubleSpinMaxX</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinMaxX">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupY">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>y-range</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioAutoCalcY">
        <property name="text">
         <string>auto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioManualY">
        <property name="text">
         <string>manual adjustment:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>from</string>
          </property>
          <property name="buddy">
           <cstring>doubleSpinMinY</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinMinY">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>to</string>
          </property>
          <property name="buddy">
           <cstring>doubleSpinMaxY</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinMaxY">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>116</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioAutoCalcX</tabstop>
  <tabstop>radioManualX</tabstop>
  <tabstop>doubleSpinMinX</tabstop>
  <tabstop>doubleSpinMaxX</tabstop>
  <tabstop>radioAutoCalcY</tabstop>
  <tabstop>radioManualY</tabstop>
  <tabstop>doubleSpinMinY</tabstop>
  <tabstop>doubleSpinMaxY</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog1DScale</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog1DScale</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioManualX</sender>
   <signal>clicked(bool)</signal>
   <receiver>doubleSpinMinX</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioManualX</sender>
   <signal>clicked(bool)</signal>
   <receiver>doubleSpinMaxX</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioManualY</sender>
   <signal>clicked(bool)</signal>
   <receiver>doubleSpinMinY</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioManualY</sender>
   <signal>clicked(bool)</signal>
   <receiver>doubleSpinMaxY</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioAutoCalcX</sender>
   <signal>clicked(bool)</signal>
   <receiver>doubleSpinMinX</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioAutoCalcX</sender>
   <signal>clicked(bool)</signal>
   <receiver>doubleSpinMaxX</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioAutoCalcY</sender>
   <signal>clicked(bool)</signal>
   <receiver>doubleSpinMinY</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioAutoCalcY</sender>
   <signal>clicked(bool)</signal>
   <receiver>doubleSpinMaxY</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
